pkg/gitops/flux/installer.go

```diff
@@ -97,15 +96,29 @@ subjects:
 
 // InstallOpts are the installation options for Flux
 type InstallOpts struct {
-	GitOptions           git.Options
-	GitPaths             []string
-	GitLabel             string
-	GitFluxPath          string
-	GitPrivateSSHKeyPath string
-	Namespace            string
-	Timeout              time.Duration
-	Amend                bool
-	WithHelm             bool
+	GitOptions  git.Options
+	GitPaths    []string
+	GitLabel    string
+	GitFluxPath string
+	Namespace   string
+	Timeout     time.Duration
+	Amend       bool // TODO: remove, as we eventually no longer want to support this mode?
+	WithHelm    bool
+}
+
+// CopyFrom copies the values from the provided InstallOpts into this InstallOpts.
+func (o *InstallOpts) CopyFrom(that *InstallOpts) {
```

Comment:
What is this for?

Reply:
These are remnants of the logic I had to merge settings retrieved from CLI and the ones retrieved from `ClusterConfig`. I didn't clean everything up as I thought I'd either re-use these or delete them if it turns out I don't need them anymore when we add support for configuration via `ClusterConfig`. Let me know if you have a strong opinion about this, and I can clean-up more "eagerly".

Reply:
I removed them in the end, in a dedicated commit, which I will not squash when merging, so that it can easily be reverted should this be required.